src/components/milvusCookieConsent/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import CookieConsent from "react-cookie-consent";
+import * as styles from "./milvusCookieConsent.module.less";
+import { useLocation } from "@reach/router";
+import { initializeAndTrack } from "gatsby-plugin-gdpr-cookies";
+
+function isBrowser() {
+  return typeof window !== "undefined";
+}
+
+function getValue(key, defaultValue) {
+  return isBrowser() && window.localStorage.getItem(key)
+    ? JSON.parse(window.localStorage.getItem(key))
+    : defaultValue;
+}
+
+function setValue(key, value) {
+  window.localStorage.setItem(key, JSON.stringify(value));
+}
+
+function useStickyState(defaultValue, key) {
+  const [value, setter] = React.useState(() => {
+    return getValue(key, defaultValue);
+  });
+
+  React.useEffect(() => {
+    setValue(key, value);
+  }, [key, value]);
+
+  return [value, setter];
+}
+
+const MilvusCookieConsent = () => {
+  const location = useLocation();
+  if (isBrowser()) {
+    initializeAndTrack(location);
+  }
+
+  const [bannerHidden, setBannerHidden] = useStickyState(
+    false,
+    "consentCookieHidden"
+  );
+
+  const EnableAnalytics = () => {
```

Comment:
The first letter of the normal function name should be lowercase. Only react function component or class is capital.